main.go

```diff
@@ -69,6 +69,8 @@ func newDefragCommand() *cobra.Command {
 
 	defragCmd.Flags().BoolVar(&globalCfg.dryRun, "dry-run", viper.GetBool("dry-run"), "evaluate whether or not endpoints require defragmentation, but don't actually perform it")
 
+	defragCmd.Flags().BoolVar(&globalCfg.skipHealthcheckClusterEndpoints, "skip-healthcheck-cluster-endpoints", viper.GetBool("skip-healthcheck-cluster-endpoints"), "skip cluster endpoint discovery during health check and only check the endpoints provided via --endpoints")
```

Comment:
This flag should be exclusive with `--cluster` and `--discovery-srv`

Reply:
addressed in latest commit